tools/rollup-es3/src/es3/Es3Tokens.ts

```diff
@@ -153,11 +153,11 @@ export const defaultEs3CheckTokens:IEs3CheckKeyword[] = [
         ]  
     },
     {
-        funcNames: [ /([\w0-9]*)\.toISOString[\s]*\(/g ],
+        funcNames: [ /([\w0-9\$]*)\.toISOString[\s]*\(/g ],
         errorMsg: "[%funcName%] is not supported in an ES3 environment, use CoreUtils.toISOString()",
         ignoreFuncMatch: [ 
-            "CoreUtils.toISOString",         // Make sure this isn't a reference to CoreUtils.isISOString()
-            "Utils.toISOString"              // or if it's a reference to Utils.isISOString()
+            /CoreUtils(\$[\d]+)+\.toISOString/,     // Make sure this isn't a reference to CoreUtils.isISOString(); CoreUtils$1.isISOString();
```

Comment:
Updating this to support an issue that Xiao was having while attempt to update the version of TypeScript which caused the CoreUtils reference to become CoreUtils$0.toISOString()